release.nix

```diff
@@ -1,34 +1,53 @@
 let
   pkgs = import ./nixpkgs.nix;
 
-  dhall-kubernetes-smoketests =
-    pkgs.runCommand
-      "all-generated-files-compile"
-      { nativeBuildInputs = [
-          pkgs.python3
-          pkgs.bash
-          pkgs.dhall
-          pkgs.glibcLocales
-        ];
-      }
+  nativeBuildInputs = [
+    pkgs.git
+    pkgs.python3
+    pkgs.bash
+    pkgs.dhall
+    pkgs.dhall-json
+    pkgs.dhall-text
+    pkgs.glibcLocales
+  ];
+
+  runCommand = name: script:
+    pkgs.runCommand name { inherit nativeBuildInputs; } script;
+
+  generatedFilesCompile =
+    runCommand
+      "generated-files-compile"
       ''
         cd ${./.}
-        LC_ALL=en_US.UTF-8 python3 ${./check-source.py}
+        LC_ALL=en_US.UTF-8 ./scripts/check-source.py
 
         touch $out
       '';
 
-  # Derivation that trivially depends on the current directory so that Hydra's
-  # pull request builder always posts a GitHub status on each revision
-  pwd = pkgs.runCommand "pwd" { here = ./.; } "touch $out";
```

Comment:
We should not remove this one, otherwise Hydra cannot approve the build (as you can see from the forever-pending build status on the PR)

Reply:
Yes indeed, the status build status did not work. But the reason was that I removed the `releaseTools.aggregate` which does some Hydra magic.

Reply:
Yep, Hydra needs a derivation that starts with "dhall-" to push the status back to github (config should be [here](https://github.com/dhall-lang/dhall-lang/blob/master/nixops/logical.nix)).

Also note that the `pwd` derivation was there such that if files that are not included in any derivation change then Hydra is not going to post back any status. I think we'll be fine, but we might have to reintroduce it if we stumble on this again.